google-cloud-bigtable/lib/google/cloud/bigtable/flat_row.rb

```diff
@@ -17,14 +42,18 @@ def initialize family, qualifier, timestamp_micros, value, labels = []
             @labels = labels
           end
 
-          # timestamp in milliseconds
-          # @return [Integer]
+          # Timestamp in "Time" object format
+          # @return [Time | nil]
 
           def timestamp
-            Time.at(@timestamp / 100_0000)
+            Time.at(@timestamp_micros / 100_0000) if @timestamp_micros
           end
 
-          def == other # :nodoc:
+          # Cell object comparator
+          #
+          # @return [Boolean]
+
+          def == other
```

Comment:
This method was :nodoc: before, should it be @private now?

Reply:
Added private